extensions/wikia/Chat2/Chat.class.php

```diff
@@ -157,7 +157,7 @@ public static function banUser( $subjectUserName, User $adminUser, $time, $reaso
 	 * @return bool
 	 * @throws DBUnexpectedError
 	 */
-	public static function blockPrivate( $subjectUserName, $dir = self::PRIVATE_BLOCK_ADD, $requestingUser ) {
+	public static function blockPrivate( $subjectUserName, $requestingUser, $dir = self::PRIVATE_BLOCK_ADD ) {
```

Comment:
Is this used anywhere else than https://github.com/Wikia/app/pull/11237/files/ca62d80bfee1ee1231281e1d396eb937ec0f2c22..b56fb15ae7225111f3676a2471460ff7075f9dd4#diff-8b4518bf8d9921aa975480c104a8173bR199 ?